cmds/core/wc/wc.go

```diff
@@ -105,7 +115,8 @@ func (c *cmd) run() error {
 	}
 
 	for _, v := range c.args {
-		f, err := os.Open(v)
```

Comment:
I much prefer line 108 as it is.

Reply:
To some extent, for some changes, it looks like you are applying C conventions to Go.